README.md

```diff
@@ -10,6 +10,8 @@ specification language.
 It also holds the WebAssembly testsuite, which tests numerous aspects of
 conformance to the spec.
 
+View the spec at [webassembly.github.io/spec](webassembly.github.io/spec/).
```

Comment:
This is going to be a broken link. Without a scheme, it gets treated as a relative link to:

https://github.com/WebAssembly/spec/blob/master/webassembly.github.io/spec

To fix that, you should add a scheme, preferably https://.